app/javascript/mastodon/features/compose/components/language_dropdown.js

```diff
@@ -37,8 +33,8 @@ class LanguageDropdownMenu extends React.PureComponent {
   };
 
   state = {
-    mounted: false,
     searchValue: '',
+    placement: 'bottom',
```

Comment:
Shouldn't this be in `LanguageDropdown` instead?

Reply:
Yes, that's correct! I've fixed this now.